frontend/src/views/HomeView.vue

```diff
@@ -1 +1,68 @@
-<template><h1>Test</h1></template>
+<template>
+    <div class="mb-4">
+      <ElButton
+        key="primary"
+        type="text"
+        @click="showConfirmationDialog(roleId)"
+        style="color: #29517A;"
```

Comment:
Please, don't use inline styling, use classes. All the colors you need, are in `colors.scss` file